esp/logging/loggingagent/cassandraloggingagent/cassandralogagent.cpp

```diff
@@ -163,10 +154,17 @@ void CCassandraLogAgent::queryTransactionSeed(const char* appName, StringBuffer&
     cassSession->disconnect();
 }
 
-void CCassandraLogAgent::setKeySpace(const char *keyspace)
+void CCassandraLogAgent::setSessionOptions(const char *keyspace)
 {
     StringArray opts;
-    setCassandraLogAgentOption(opts, "keyspace", keyspace);
+    setCassandraLogAgentOption(opts, "contact_points", dbServer.str());
+    if (!dbUserID.isEmpty() && !dbPassword.isEmpty())
```

Comment:
I've struggled with this in the past because it's conceivable that the UserID could have a blank password. I'm not sure if that's allowed in cassandra, but if it is, we might want to allow it